dev/MRTCore/SignConfig.xml

```diff
@@ -13,6 +13,10 @@
    <file src="__INPATHROOT__\x86\mrm.dll" signType="AuthenticodeFormer" /> 
    <file src="__INPATHROOT__\x86\Microsoft.ApplicationModel.Resources.dll"  signType="AuthenticodeFormer" /> 
  </job>
+ <job platform="ARM" certSubject="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" configuration="Release" dest="__INPATHROOT__\ARM\signed" jobname="MRT Core binary sign (ARM)" approvers="">
```

Comment:
we are going to use the proejct reunion signconfig I'm pretty sure ...

Reply:
until it's integrated, I'd still update the MRT one